tycho-its/src/test/java/org/eclipse/tycho/test/apitools/ApiToolsTest.java

```diff
@@ -47,16 +47,15 @@ public void testVerify() throws Exception {
 		Verifier verifier = getVerifier("api-tools", true, true);
 		File repo = ResourceUtil.resolveTestResource("repositories/api-tools");
 		verifier.addCliOption("-DbaselineRepo=" + repo.toURI());
-		try {
-			verifier.executeGoals(List.of("clean", "verify"));
-			fail("No API errors where detected!");
-		} catch (VerificationException e) {
-			verifier.verifyTextInLog("1 API ERRORS");
-			verifier.verifyTextInLog("0 API warnings");
-			verifier.verifyTextInLog("The type bundle.InterfaceA has been removed from api-bundle");
-			verifier.verifyTextInLog("The type bundle.ClassA has been removed from api-bundle");
-			// TODO: check with api-filter
-			// TODO: check with second plugin with BREE?
-		}
+
+		assertThrows(VerificationException.class, () -> verifier.executeGoals(List.of("clean", "verify")));
```

Comment:
The assert should contain a message why this is expected.

Reply:
Done.